projects/components/src/label-tag/label-tag.component.ts

```diff
@@ -0,0 +1,28 @@
+import { ChangeDetectionStrategy, Component, Input, OnInit } from '@angular/core';
+import { Colors } from '../../../common/src/color/colors';
+
+@Component({
+  selector: 'ht-label-tag',
+  styleUrls: ['./label-tag.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="label-tag" [ngStyle]="{ backgroundColor: this.backgroundColor, color: this.labelColor }">
+      {{ this.label }}
+    </div>
+  `
+})
+export class LabelTagComponent implements OnInit {
+  @Input()
+  public label?: string;
+
+  @Input()
+  public labelColor?: string;
+
+  @Input()
+  public backgroundColor?: string;
+
+  public ngOnInit() {
+    this.backgroundColor = this.backgroundColor ?? Colors.Gray1;
```

Comment:
I think the `ngStyle` for that property does not get applied, if the value is undefined, so probably we can define these defaults directly as CSS

Reply:
makes sense.